src/supertux/sector.hpp

```diff
@@ -187,6 +188,16 @@ class Sector
   float get_height() const;
 
   /**
+   * returns the editor size (in tiles) of a sector
+   */
+  Size get_editor_size() const;
+
+  /**
+   * resize all tilemaps with given size
+   */
+  void resize_sector(Size old_size, Size new_size);
```

Comment:
Just out of curiosity: Can you pass by ref?

const Size& old_size, const Size& new_size